drivers/hmis/app/models/hmis/hud/organization.rb

```diff
@@ -31,6 +31,14 @@ class Hmis::Hud::Organization < Hmis::Hud::Base
     where(id: ids, data_source_id: user.hmis_data_source_id)
   end
 
+  scope :matching_search_term, ->(search_term) do
+    return none unless search_term.present?
+
+    search_term.strip!
+    query = "%#{search_term.split(/\W+/).join('%')}%"
+    where(o_t[:OrganizationName].matches(query).or(o_t[:id].eq(search_term)).or(o_t[:organization_id].eq(search_term)))
```

Comment:
Just to check, should we search by OrganizationCommonName here too? I noticed Projects also has a CommonName field, but doesn't filter by it [here](https://github.com/greenriver/hmis-warehouse/blob/29f8a66c9cbc1cb8f102c30023adc7ae02740b3d/drivers/hmis/app/models/hmis/hud/project.rb#L135), which is why I did not add that here.

Reply:
Good Q, no need to search the common name field. That's from the HUD CSV spec but we don't really use it.